src/SfxWeb/src/app/modules/event-store/option-picker/option-picker.component.html

```diff
@@ -0,0 +1,11 @@
+<app-collapse-container collapsed="true" sectionName="Select types">
+	<h5 collapse-header>
+		Select event types
+	</h5>
+	<div collapse-body data-cy="option-picker">
+		<div *ngFor="let option of options" style="display: inline-block; padding-right: 15px;">
```

Comment:
dont use inline style here, make the parent div, the one with collapse-body use a class like

.options-wrapper{
display: flex;
flex-direction: row;
flex-wrap: wrap;
}

.options-wrapper{ > * {
padding-right: 15px;
}